source/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,118 @@
+=================
+Update a Document
+=================
+
+.. default-domain:: mongodb
+
+You can update a single document using the ``updateOne()`` method on
+a ``MongoCollection`` object. The method accepts a **filter** that matches the
+document you want to update and an **update** statement that instructs the
+driver to change the matching document. The ``updateOne()`` method only
+updates the first document that matches the filter.
+
+To perform an update with the ``updateOne()`` method, you must pass
+a query filter and an update document. The query filter specifies the criteria
+for which document to perform the update on and the update document provides
+instructions on what changes to make to it.
+
+You can optionally pass an instance of ``UpdateOptions`` to the method in
+order to specify the behavior. For example, if you set the ``upsert`` field of
+the ``UpdateOptions`` object to ``true``, the operation inserts a new
+document from the update document if no documents match the query filter.
```

Comment:
You should check but I think it actually combines the filter with the update document.  So in the example you'd end up with a document that also contains `"title":  "Cool Runnings"`

Reply:
You're right, will update.

Reply:
Looks like you removed UpdateOptions entirely.  Is that the intention for all driver docs going forward?